frontend/amundsen_application/models/issue_results.py

```diff
@@ -9,18 +9,30 @@ class IssueResults:
     def __init__(self,
                  issues: List[DataIssue],
                  total: int,
-                 all_issues_url: str) -> None:
+                 open_count: int,
+                 all_issues_url: str,
+                 open_issues_url: str,
+                 closed_issues_url: str) -> None:
```

Comment:
1. keep the previous order for compatibility. 
2. make your new args optional (with Optional[type] annotation and default value)